sentinel-demo/sentinel-demo-okhttp/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>sentinel-demo</artifactId>
+        <groupId>com.alibaba.csp</groupId>
+        <version>1.8.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>sentinel-demo-okhttp</artifactId>
+
+    <properties>
+        <spring.boot.version>2.1.3.RELEASE</spring.boot.version>
+        <okhttp.version>3.6.0</okhttp.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.alibaba.csp</groupId>
+            <artifactId>sentinel-core</artifactId>
+        </dependency>
```

Comment:
This can be omitted in demo, since `sentinel-okhttp-adapter` has the dependency. Adding it is all right, but new user will feel uncertain if this is is must when he use this `adapter`.

Reply:
恩 修复了 之前还奇怪 idea为啥报警告呢
![image](https://user-images.githubusercontent.com/10150229/81374392-bab3ec00-9131-11ea-9545-1592326051b6.png)
稍后 我在其他pr 把其他demo 类似也一起fix了